apps/web/modules/survey/survey-editor/lib/project.ts

```diff
@@ -0,0 +1,96 @@
+import { Language, Prisma, Project } from "@prisma/client";
+import { cache as reactCache } from "react";
+import { z } from "zod";
+import { prisma } from "@formbricks/database";
+import { cache } from "@formbricks/lib/cache";
+import { projectCache } from "@formbricks/lib/project/cache";
+import { validateInputs } from "@formbricks/lib/utils/validate";
+import { DatabaseError, ResourceNotFoundError } from "@formbricks/types/errors";
+
+export const getProjectByEnvironmentId = reactCache(
+  async (environmentId: string): Promise<Project | null> =>
+    cache(
+      async () => {
+        validateInputs([environmentId, z.string().cuid2()]);
+
+        let projectPrisma;
```

Comment:
I think there isn't any need to declare this variable first with a `let` keyword, doing so also leads to `projectPrisma` being defined as `any` type (hovering over it shows that its `any`). This is an issue with all the services in `lib/project.ts` files in the survey module. Please remove this `let` variable and just use a `const` directly